sap/camel-sap-starter/pom.xml

```diff
@@ -1,181 +1,245 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.camel.springboot</groupId>
+    <artifactId>camel-sap-starter-parent</artifactId>
+    <version>3.20.1-SNAPSHOT</version>
+  </parent>
+  <groupId>org.fusesource</groupId>
```

Comment:
is this groupId correct? shouldn't it be org.apache.camel.springboot?

Reply:
Hi @Croway ! Thanks for reviewing! 

I haven't changed the groupId in this PR. I think @cunningt set it to `org.fusesource` rather than `apache` is because of licensing issues with `SAP` hence not part of Apache which would explain the groupId change...

Reply:
okok, makes sense, anyway, this component is not created by apache, I wonder if org.fusesource makes sense in 2023 then.. what about com.redhat?

Reply:
For sure that can be changed but I don't think it's in the scope of this PR. We would have to start changing it in camel-sap